### ISSUE TITLE
I suppress sema-duplicated-attrname, but this doesn't disable the recoloring

### ISSUE THREAD

**Issue by stereomato** [nix-community/nixd#619]
**Describe the bug**
See title. I expect suppressing that to also disable the recoloring of text.

**Logs (Required)**

```
E[13:41:30.728] 22081: nixos eval expr: -32001: attribute '[35;1mservices.xserver.desktopManager.gnome.enable[0m' already defined at [35;1m/nix/store/xy5mc6jlsqd3m2zdhkx90j9h7zw51xnb-source/machines/common/desktop-environment.nix:47:2[0m
I[13:41:30.763] 22081: <-- reply(3)
I[13:41:30.763] 22081: --> notify $/progress
E[13:41:30.763] 22081: nixos eval expr: -32001: attribute '[35;1mservices.xserver.desktopManager.gnome.enable[0m' already defined at [35;1m/nix/store/xy5mc6jlsqd3m2zdhkx90j9h7zw51xnb-source/machines/common/desktop-environment.nix:47:2[0m
I[13:41:30.946] 22081: <-- textDocument/didOpen

```

**Configuration**

```
"nixd": {
            "formatting": {
                // This is the default if ommited.
                "command": [ "nixfmt" ]
            },
            "options": {
                // By default, this entriy will be read from `import <nixpkgs> { }`
                // You can write arbitary nix expression here, to produce valid "options" declaration result.
                // Tip: for flake-based configuration, utilize `builtins.getFlake`
                "nixos": {
                    "expr": "(builtins.getFlake \"/home/stereomato/Documents/Software Development/Repositories/Personal/nixos-setup\").nixosConfigurations.Taihou.options"
                },
                //"home-manager": {
                //    "expr": "(builtins.getFlake \"/home/stereomato/Documents/Software Development/Repositories/Personal/nixos-setup\").homeConfigurations.\"stereomato@Taihou\".options"
                //}
            },
            "diagnostic": {
                "suppress": [
                    "sema-duplicated-attrname"
                ]
            }
        },
```

**To Reproduce**
```
services = {};
services = {};
```
That¿s enough to trigger the behavior. The 2nd services isn't the same color as the first, instead being red.

**Expected behavior**
The color is not changed when sema-duplicated-attrname is supressed

**Screenshots**
![image](https://github.com/user-attachments/assets/6e546a3e-5bec-45e9-b4f7-37efe97303bd)

**Additional context**
I just switched to `nixd` and I like what I see!

**Comment by inclyc** [comment]
Coloring stuff is fixed in 2.5.1.
See #616

**Comment by stereomato** [comment]
oof, didn't see. Thanks!
